From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Sunu Aziz Rahayu <sunuazizrahayu@users.noreply.github.com>
Date: Fri, 26 Sep 2026 00:00:00 +0700
Subject: [PATCH] vendor_interfaces: remove stale LGE vibrator 1.0

android.hardware.vibrator@1.0 sudah dihapus di Lineage 23.2 / Android
16 (AIDL), sehingga modul ini gagal Soong:

  "vendor.lge.hardware.vibrator@1.0_genc++" depends on undefined
  module "android.hardware.vibrator@1.0_interface"

Modul LGE ini tidak dibutuhkan untuk GSI generic, hapus Android.bp-nya
saja agar Soong melewatkannya (IVibratorEx.hal yatim aman diabaikan).

---
 vendor/lge/hardware/vibrator/1.0/Android.bp | 15 ---------------
 1 file changed, 15 deletions(-)
 delete mode 100644 vendor/lge/hardware/vibrator/1.0/Android.bp

diff --git a/vendor/lge/hardware/vibrator/1.0/Android.bp b/vendor/lge/hardware/vibrator/1.0/Android.bp
deleted file mode 100644
index 0000000..0000000
--- a/vendor/lge/hardware/vibrator/1.0/Android.bp
+++ /dev/null
@@ -1,15 +0,0 @@
-// This file is autogenerated by hidl-gen -Landroidbp.
-
-hidl_interface {
-    name: "vendor.lge.hardware.vibrator@1.0",
-    root: "vendor",
-    srcs: [
-        "IVibratorEx.hal",
-    ],
-    interfaces: [
-        "android.hardware.vibrator@1.0",
-        "android.hidl.base@1.0",
-    ],
-    gen_java: true,
-}
-
--
2.54.0
